docs/devel/releasing.md

```diff
@@ -81,101 +117,106 @@ oncall.
 Before proceeding to the next step:
 
 ```sh
-export BRANCHPOINT=v0.20.2-322-g974377b
+export GITHASH=v1.2.0-alpha.2.164+b44c7d79d6c9bb
+```
+
+Where `v1.2.0-alpha.2.164+b44c7d79d6c9bb` is the git hash you decided on. This
+will become your release point.
+
+### Cutting/branching the release
+
+You'll need the latest version of the releasing tools:
+
+```console
+git clone git@github.com:kubernetes/kubernetes.git
+cd kubernetes
+```
+
+or `git checkout upstream/master` from an existing repo.
+
+#### Cutting an alpha release (`vX.Y.0-alpha.W`)
+
+Figure out what version you're cutting, and
+
+```console
+export VER="vX.Y.0-alpha.W"
 ```
 
-Where `v0.20.2-322-g974377b` is the git hash you decided on. This will become
-our (retroactive) branch point.
+then, run
 
-#### Branching, Tagging and Merging
+```console
+release/cut-official-release.sh "${VER}" "${GITHASH}"
```

Comment:
Is this true?  Do we use cut-official-release.sh to cut alpha releases?  If so, how does it end up doing different things than are described below under 'Cutting and official release'?